noir/noir-repo/compiler/noirc_evaluator/src/brillig/brillig_ir/codegen_control_flow.rs

```diff
@@ -258,4 +225,167 @@ impl<F: AcirField + DebugToString> BrilligContext<F> {
             }
         });
     }
+
+    /// Computes the size of a parameter if it was flattened
```

Comment:
All of these moved from entry_point.rs, since they are not only used in the entrypoint